docs/man.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+has_children: true
+nav_order: 4
+has_toc: false
+---
+
+# Command line reference
+
+## coreos-installer
+
+### Supported commands
+
+- [coreos-installer](coreos-installer.md)
+
+- [coreos-installer-install](coreos-installer-install.md)
+- [coreos-installer-download](coreos-installer-download.md)
+- [coreos-installer-list-stream](coreos-installer-list-stream.md)
+
+- [coreos-installer-iso](coreos-installer-iso.md)
+- [coreos-installer-iso-ignition](coreos-installer-iso-ignition.md)
+- [coreos-installer-iso-ignition-embed](coreos-installer-iso-ignition-embed.md)
```

Comment:
I think these three could be just bundled into `iso-ignition.md`. Same for `pxe-ignition-*` sub-commands below.